html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Chained.pm

```diff
@@ -33,6 +33,11 @@ has_field 'authentication_source' =>
    options_method => \&options_authentication_source,
   );
 
+has_field authentication_source_can_continue => (
```

Comment:
If it is enabled then the source defined by the authentication_source attribute can be used in addition to the registered user.